db/liquibase/liquibase.properties

```diff
@@ -0,0 +1,8 @@
+driver=org.postgresql.Driver
+classpath=postgresql-42.2.5.jar
+url=jdbc:postgresql://127.0.0.1:5432/testdb
+username=avram
+password=test
+changeLogFile=changelog.xml
```

Comment:
This currently has the test db url, username and password - we can template-ize this in a future PR